### PR TITLE
Add apache_version fact

### PR DESCRIPTION
**Pull request by jyaworski**

### CODE REVIEW

**Comment by bmjen** [comment]
Thanks @jyaworski !

File: lib/facter/apache_version.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Facter.add(:apache_version) do
+  setcode do
+    if Facter::Util::Resolution.which('apachectl')
+      apache_version = Facter::Util::Resolution.exec('apachectl -v 2>&1')
```
Comment (igalic) [inline lib/facter/apache_version.rb]: on older versions of Debuntu, this will not work, because they renamed their apachectl to apache2ctl

Comment (jyaworski) [inline lib/facter/apache_version.rb]: How old is 'older'? 7?

Comment (hunner) [inline lib/facter/apache_version.rb]: It may not work, but then it will just be undef so same ol' same ol'.

Comment (igalic) [inline lib/facter/apache_version.rb]: 2.2

**Pull request merged**